modules/accountplantemplate/view/hovedbok.php

```diff
@@ -53,160 +55,202 @@
 
   </tr>
   <tr>
+    <? // Credit text ?>
     <td class="menu">Kredit tekst</td>
     <td class="<? print $account->CreditColor ?>"><input type="text" name="accountplantemplate.credittext" value="<? print $account->credittext ?>" size="50">
+    <? // Color: ?>
         Farge: <? print $_lib['form3']->Type_menu3(array('table'=>$db_table, 'field'=>'CreditColor', 'value'=>$account->CreditColor, 'type'=>'CreditColor', 'required' => 1)) ?></td>
 
   </tr>
   <tr>
+    <? // Currency ?>
     <td class="menu">Valuta</td>
     <td><? $_lib['form2']->checkbox2($db_table, "EnableCurrency", $account->EnableCurrency,'') ?> <? $_lib['form2']->currency_menu2($db_table, "Currency", $account->Currency) ?></td>
 
   </tr>
   <tr>
+    <? // Text - information ?>
     <td class="menu">Tekst - informasjon</td>
     <td><input type="text" name="accountplantemplate.Description" value="<? print $account->Description ?>" size="60"></td>
   </tr>
   <tr>
+    <? // Ledger number ?>
     <td class="menu">Reskontronummer</td>
     <td><? $_lib['form2']->checkbox2($db_table, "EnableReskontro",$account->EnableReskontro,'') ?>
     <? print $_lib['form3']->Type_menu3(array('table'=>$db_table, 'field'=>'ReskontroAccountPlanType', 'value'=>$account->ReskontroAccountPlanType, 'type'=>'AccountPlanType', 'required' => 0)) ?>
   </tr>
   <tr>
+    <? // KID reference ?>
     <td class="menu">KID referanse</td>
     <td><? $_lib['form2']->checkbox2($db_table, "EnablePostPost",$account->EnablePostPost,'') ?></td>
 
   </tr>
   <tr>
+    <? // VAT code ?>
     <td class="menu">Mva kode</td>
     <td> <? print $_lib['form3']->checkbox(array('table'=>$db_table, 'field'=>'EnableVAT', 'value'=>$account->EnableVAT)) ?> Kode <? print $_lib['form3']->vat_menu3(array('table'=>$db_table, 'field'=>'VatID', 'value'=>$account->VatID, 'vatid'=>'1')) ?> overstyres: <? $_lib['form2']->checkbox2($db_table, "EnableVATOverride",$account->EnableVATOverride,'') ?></td>
   </tr>
   <tr>
+    <? // Quantity ?>
     <td class="menu">Mengde</td>
     <td><? $_lib['form2']->checkbox2($db_table, "EnableQuantity", $account->EnableQuantity,'') ?></td>
 
   </tr>
   <tr>
+    <? // Department ?>
     <td class="menu">Avdeling</td>
     <td><? $_lib['form2']->checkbox2($db_table, "EnableDepartment",$account->EnableDepartment,'') ?> Default: <? $_lib['form2']->department_menu2(array('table' => $db_table, 'field' => 'DepartmentID', 'value' => $account->DepartmentID, 'tabindex' => $tabindex++, 'accesskey' => 'V')) ?></td>
   </tr>
   <tr>
+    <? // Project ?>
     <td class="menu">Prosjekt</td>
     <td><? $_lib['form2']->checkbox2($db_table, "EnableProject",$account->EnableProject,'') ?> Default: <? $_lib['form2']->project_menu2(array('table' => $db_table,  'field' =>  'ProjectID',  'value' =>  $account->ProjectID, 'tabindex' => $tabindex++, 'accesskey' => 'P')) ?>
   </tr>
   <tr class="result">
+    <? // Reports (that main ledger account plan should be used) ?>
```

Comment:
Reports (that main ledger account plan should be used) => Reports (that main ledger account plan should be used in)